### DISCUSSION TITLE
Welcome to novelsave Discussions!

### DISCUSSION THREAD

**Discussion by m-haisham** [m-haisham/novelsave#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tetdesu** [comment]
Uhm just a question 😂
Can The novel's Synopsis or Summary be added to the epub ? :3

**Reply by m-haisham** [reply]
All the new sources that were added recently add the synopsis to epub as tag `<description>` in metadata.

Any specific source you are curious about?

**Reply by Tetdesu** [reply]
Oh I See.
None so far.

Thanks for making this btw 😁
